editoast/core_client/src/pathfinding.rs

```diff
@@ -160,7 +160,8 @@ pub enum PathfindingNotFound {
 /// `begin` is always less than `end`.
 #[editoast_derive::openapi_schema]
 #[derive(Serialize, Deserialize, Clone, Debug, ToSchema, Hash, PartialEq, Eq)]
-pub struct TrackRange {
+#[schema(as = CoreTrackRange)]
+pub struct CoreTrackRange {
```

Comment:
```suggestion
pub struct TrackRange {
```

There's no need for a prefix, the crate name acts as it. In situations where there is a conflict or a possible confusion, let's just prefix it with the full path `core_client::TrackRange`? OpenAPI components do not have namespaces though, so the prefix's necessary there, like you did with `#[schema(as =`.

Reply:
@leovalais I made the changes

Reply:
This change requires to use `#[schema(value_type = Vec<CoreTrackRange>)]`? If so, I'm not a fan, as it makes it easy to forget / error prone.

Reply:
After discussing with @flomonster and @woshilapin, let's keep the `#[schema()]` if you're fine with it.

Reply:
I'm good with the `#[schema()]` 👍